deploy/database/data.button.sql

```diff
@@ -1152,3 +1152,315 @@ UPDATE button
 SET flavor_text='This button gets a different random recipe in each game, with an anime formula'
 WHERE name='RandomBMAnime';
 
+UPDATE button SET flavor_text="Rikachu is a bootleg Button Men created at Origins. He's a Pokedog - from the Pokeman ad opposite the Tess page." WHERE name="Rikachu";
+UPDATE button SET flavor_text="ConMan plays games at conventions until (and sometimes past) dawn, drinks coffee like it's water, and he likes to beat people up." WHERE name="ConMan";
+UPDATE button SET flavor_text="The Cheese Weasel is a clever and cunning rodent, bent on quiet world domination. He likes cheese (duh!), any opportunity for eating cheese, stealing cheese...and beating people up if they attempt to come between him and his cheese." WHERE name="Cheese Weasel";
+UPDATE button SET flavor_text="Bennett is a big strong boy who loves to take his time. He enjoys math, videotapes, and beating people up." WHERE name="Bennett";
+UPDATE button SET flavor_text="Chris knows what she likes and she knows how to get it. She likes chemistry, going to the park, and beating people up." WHERE name="Chris";
+UPDATE button SET flavor_text="Darwin is young entrepreneur who enjoys politics, French, and beating people up." WHERE name="Darwin";
+UPDATE button SET flavor_text="Hale is a considerate fellow and excels in every class. He is never too busy to stop and beat people up." WHERE name="Hale";
+UPDATE button SET flavor_text="Morgan knows the rules. She never does anything without reason, and she loves to beat people up." WHERE name="Morgan";
```

Comment:
Nit: i'd say "she never does anything without *a* reason", though the way it's written isn't strictly unparseable.